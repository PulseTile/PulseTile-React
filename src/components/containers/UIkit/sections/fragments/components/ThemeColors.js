import React from 'react';

/**
 * This component returns content of ThemeColors section
 */
const ThemeColors = () => {
  return (
    <div id="palette-color" className="ui-section">
      <strong className="ui-title">Color Palette for Themes</strong>
      <div className="form-group-wrapper">
        <div className="row">
          <div className="col-xs-6 col-sm-4">
            <div className="form-group">
              <div className="palette-color">
                <span className="palette-color-icon" style={{ background: 'rgb(13, 103, 47)' }}></span>
                <span className="palette-color-name ng-binding">Green Theme</span>
              </div>
            </div>
          </div>
          <div className="col-xs-6 col-sm-4">
            <div className="form-group">
              <div className="palette-color">
                <span className="palette-color-icon" style={{ background: 'red' }}></span>
                <span className="palette-color-name ng-binding">Red Theme</span>
              </div>
            </div>
          </div>
          <div className="col-xs-6 col-sm-4">
            <div className="form-group">
              <div className="palette-color">
                <span className="palette-color-icon" style={{ background: 'blue' }}></span>
                <span className="palette-color-name ng-binding">Blue Theme</span>
              </div>
            </div>
          </div>
          <div className="col-xs-6 col-sm-4">
            <div className="form-group">
              <div className="palette-color">
                <span className="palette-color-icon" style={{ background: 'brown' }}></span>
                <span className="palette-color-name ng-binding">Brown Theme</span>
              </div>
            </div>
          </div>
          <div className="col-xs-6 col-sm-4">
            <div className="form-group">
              <div className="palette-color">
                <span className="palette-color-icon" style={{ background: 'yellow' }}></span>
                <span className="palette-color-name ng-binding">Yellow Theme</span>
              </div>
            </div>
          </div>
          <div className="col-xs-6 col-sm-4">
            <div className="form-group">
              <div className="palette-color">
                <span className="palette-color-icon" style={{ background: 'pink' }}></span>
                <span className="palette-color-name ng-binding">Pink Theme</span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default ThemeColors;
