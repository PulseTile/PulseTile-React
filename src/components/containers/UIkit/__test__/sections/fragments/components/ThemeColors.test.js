import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import ThemeColors from '../../../../sections/fragments/components/ThemeColors';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <ThemeColors />', () => {
  it('should renders ThemeColors with props correctly', () => {
    const component = shallow(<ThemeColors />);
    expect(component).toMatchSnapshot();
  });
});
