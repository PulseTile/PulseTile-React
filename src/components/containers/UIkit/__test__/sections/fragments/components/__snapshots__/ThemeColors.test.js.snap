// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <ThemeColors /> should renders ThemeColors with props correctly 1`] = `
<div
  className="ui-section"
  id="palette-color"
>
  <strong
    className="ui-title"
  >
    Color Palette for Themes
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-6 col-sm-4"
      >
        <div
          className="form-group"
        >
          <div
            className="palette-color"
          >
            <span
              className="palette-color-icon"
              style={
                Object {
                  "background": "rgb(13, 103, 47)",
                }
              }
            />
            <span
              className="palette-color-name ng-binding"
            >
              Green Theme
            </span>
          </div>
        </div>
      </div>
      <div
        className="col-xs-6 col-sm-4"
      >
        <div
          className="form-group"
        >
          <div
            className="palette-color"
          >
            <span
              className="palette-color-icon"
              style={
                Object {
                  "background": "red",
                }
              }
            />
            <span
              className="palette-color-name ng-binding"
            >
              Red Theme
            </span>
          </div>
        </div>
      </div>
      <div
        className="col-xs-6 col-sm-4"
      >
        <div
          className="form-group"
        >
          <div
            className="palette-color"
          >
            <span
              className="palette-color-icon"
              style={
                Object {
                  "background": "blue",
                }
              }
            />
            <span
              className="palette-color-name ng-binding"
            >
              Blue Theme
            </span>
          </div>
        </div>
      </div>
      <div
        className="col-xs-6 col-sm-4"
      >
        <div
          className="form-group"
        >
          <div
            className="palette-color"
          >
            <span
              className="palette-color-icon"
              style={
                Object {
                  "background": "brown",
                }
              }
            />
            <span
              className="palette-color-name ng-binding"
            >
              Brown Theme
            </span>
          </div>
        </div>
      </div>
      <div
        className="col-xs-6 col-sm-4"
      >
        <div
          className="form-group"
        >
          <div
            className="palette-color"
          >
            <span
              className="palette-color-icon"
              style={
                Object {
                  "background": "yellow",
                }
              }
            />
            <span
              className="palette-color-name ng-binding"
            >
              Yellow Theme
            </span>
          </div>
        </div>
      </div>
      <div
        className="col-xs-6 col-sm-4"
      >
        <div
          className="form-group"
        >
          <div
            className="palette-color"
          >
            <span
              className="palette-color-icon"
              style={
                Object {
                  "background": "pink",
                }
              }
            />
            <span
              className="palette-color-name ng-binding"
            >
              Pink Theme
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
